compiler/testData/debug/localVariables/tryFinally9.kt

```diff
@@ -0,0 +1,54 @@
+// FILE: test.kt
+
+inline fun f(block: () -> Unit) {
+    block()
+}
+
+var x: String = ""
+
+fun compute(): String {
+    try {
+        try {
+            val y = 42
+            for (i in 0 until 1) {
+                return "NORMAL_RETURN"
+            }
+        } finally {
+            var s = "NOPE"
+            x = s
+            f {
+                return "NON_LOCAL_RETURN"
+            }
+        }
+    } finally {
+        val s2 = "OK"
+        x = s2  // TODO: Why is `s2` not visible here?
```

Comment:
This is a little puzzling. I would like to have a look at that separately from this change.